packages/riipen-ui/src/components/RadioButton.jsx

```diff
@@ -0,0 +1,135 @@
+import clsx from "clsx";
+import PropTypes from "prop-types";
+import React from "react";
+
+import ThemeContext from "../styles/ThemeContext";
+import withClasses from "../utils/withClasses";
+
+import Typography from "./Typography";
+
+const RadioButton = ({
+  checked,
+  classes,
+  disabled,
+  label,
+  prefix: Prefix,
+  prefixProps,
+  suffix: Suffix,
+  suffixProps,
+  ...other
+}) => {
+  const theme = React.useContext(ThemeContext);
+
+  return (
+    <div
+      className={clsx(
+        classes,
+        "root",
+        checked && "checked",
+        disabled && "disabled"
+      )}
+    >
+      <label
+        className={clsx(checked && "checked", disabled && "disabled")}
+        htmlFor={other.id}
```

Comment:
since we're doing `other.id`, I think we should add `id` to propTypes + destructure it out of props, and just use `id`